e2e-tests/tests/smart-update/01-create-cluster.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
Technically we could have `gr-smart-update` test also, because there is a difference on how we get primary node in async and in group replication. But I'm not sure if this would be too much. Also if group replication is the new default I think by default we should add group replication tests.

Reply:
I think it would be too much, gut I agree we should have it by default for GR rather than async.
@hors @egegunes wdyt?

Reply:
we need to have both, but we can do it via separate task